packages/pigeon/lib/functional.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Why no unit tests for this file? These seem trivially unit testable, rather than having to find issues here via much-higher-level failures.

Reply:
file removed